doc/docs-revamp/dev-onboarding-quick-setup.md

```diff
@@ -0,0 +1,375 @@
+
+This guide's objective is to help you set up your development environment, test it, and write your first smart contract within an hour. This will be a simple vesting contract in which someone sends a gift of ada to a beneficiary. The beneficiary can access their gift only after a specific deadline has passed and the beneficiary has signed the contract. 
+
+## Setting up and testing your development environment
+
+### Prerequisites
+
+#### Hardware and OS requirements
+
+**Processing power**
+- Minimum: Intel Core i5 or AMD equivalent
+- Recommended: Intel Core i7 or AMD Ryzen 7 
+
+**RAM**
+- Minimum: 8GB RAM
+- Recommended: 16GB RAM or more
+
+**Operating system**
+- Linux: A recent distribution of Linux such as Ubuntu 20.04 LTS or later is preferred due to its widespread use in the Haskell community and its compatibility with Plutus tooling.
+- macOS: macOS Catalina (10.15) or later
+- Windows: While Windows can be used for Haskell development, it is less common and might require additional setup such as using Windows Subsystem for Linux (WSL2) to provide a Linux-compatible environment.
+
+### Tools
+
+To start developing with Haskell and Plutus, you'll need to set up your development environment with the essential tools. This section will guide you through installing and configuring the necessary components. 
+
+#### Nix
+
+Nix is a package manager that provides reproducible and reliable software environments. It is particularly useful in Plutus development for managing dependencies and ensuring consistent development environments across different machines. 
+
+While not strictly required, Nix is highly recommended because of its effective management of complex library dependencies. 
+
+To install Nix, follow the instructions on the [official Nix website](https://nixos.org/download.html). 
+After installing, configure Nix to use IOHK’s binary caches to speed up the build process. For important details and context, see these two documents: 
+- [Contributing to Plutus > Installing and setting up Nix](https://github.com/IntersectMBO/plutus/blob/master/CONTRIBUTING.adoc),
+- [Nix setup guide](https://github.com/input-output-hk/iogx/blob/main/doc/nix-setup-guide.md): instructions for installing and configuring Nix to work with projects at IOG.
+
+#### Haskell tool stack ("Stack")
+
+Stack is a cross-platform program for developing Haskell projects. It handles dependency management, building, and testing your Haskell code. 
+
+To install the Stack: 
+1. Visit the [official Haskell Stack website](https://docs.haskellstack.org/). 
+2. Follow the installation instructions for your operating system (Linux, macOS, or Windows).
+3. Verify your installation by running `stack --version` in your terminal.
+
+#### GHC (Glasgow Haskell Compiler)
+
+- GHC is the main compiler for Haskell with many extensions and optimizations. 
+- To install GHC, run `stack setup`. Stack will automatically install the appropriate version of GHC for your project. 
```

Comment:
I don't think we need to teach people how to install Stack, GHC and Cabal, since they are standard Haskell tooling. Also, if you are using Nix or Docker, they will already be installed for you.

I also don't think "stack setup" is the standard way of installing GHC.

Reply:
Good to know. Taking a step back to look at the bigger picture, we may need to revisit the question of how much detail is appropriate to include given our intended audience. I think I'll make a revised outline for review and discussion to simplify getting agreement about scope and detail.